src/TeamRep.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+function TeamRep({ repName, srcPhoto, profession, story, motto, jobDescription, twitterProfile, facebookProfile, githubProfile }) {
+    return (
+        <React.Fragment>
+            <div className="team">
+                <div className="card-container">
+                    <div className="card">
+                        <div className="front">
+                            <div className="cover">
+                                <img src={srcPhoto} />
+                            </div>
+                            <div className="user">
+                                <img className="img-circle" src={srcPhoto} />
```

Comment:
Codacy found an issue: [img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6058781)